dev-python/wand/wand-0.6.13.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="The ctypes-based simple ImageMagick binding for Python"
+DESCRIPTION="Wand is a ctypes-based simple ImageMagick binding for Python, supporting 2.7, 3.3+, \
+and PyPy. All functionalities of MagickWand API are implemented in Wand."
+HOMEPAGE="https://wand-py.org/"
+COPYRIGHT="2011-2018 by Hong Minhee
+	2019-2023 by E. McConville"
+LICENSE="MIT"
+REVISION="1"
+pypiVersion="59/d5/1bdd7c9662d5e9078e25ba0eb69bdb122859295746d40ab8dfef3a7b4d42"
```

Comment:
BTW: Is there a reason to keep this separately from SOURCE_URI when it is only used there? Is the idea that the actual SOURCE_URI doesn't change on upgrades?